codesydney/codesydneywebsite
# add new volunteer's profiles

## Rationale

What does this pull request do?

## Advice for Reviewers & Testing Notes

- Explain how your change should be reviewed, and where reviewers should direct their attention.

1.   Add volunteers' profile of Paolo, Praveen, Henry and Luke in volunteer page.
2.   Add avatar imgs of above volunteers into public folder.
3.   Change a tag for linkedin link to open volunteer's linkedin page in a new tag.

## Screenshots:


![New volunteer profiles](https://user-images.githubusercontent.com/64076077/157656769-0b5fa599-d3f6-4514-ac1e-8745c7b2b487.png)

## Issue Resolved

*Note - This allows the issue to be automatically linked to the PR once the PR is submitted for review*
- Fixes #24
